scripts/generatePlatformSdk.sh

```diff
@@ -65,7 +126,14 @@ pnpm exec -- \
         --filter ./packages/foundry \
         --filter ./packages/internal.foundry \
         --filter="./packages/foundry.*" \
-        --filter="./packages/internal.foundry.*"
+        --filter="./packages/internal.foundry.*" \
+
+if [[ "${PREFIX}" == "all" || "${PREFIX}" == "gotham" ]]; then
+  pnpm exec -- \
+      turbo run --output-logs=errors-only fix-lint \
+          --filter ./packages/gotham \
+          --filter="./packages/gotham.*"
+fi
```

Comment:
It's probably fine to run fix lints on both gotham and foundry every time, just for cleaner code purposes